rbac/server/api/packs.py

```diff
@@ -56,6 +90,25 @@
 
 
 @PACKS_BP.post("api/packs")
+@doc.summary("API Endpoint to create a new pack")
+@doc.description("API Endpoint to create a new pack.")
+@doc.consumes(
+    doc.JsonBody(
+        {"description": str, "name": str, "owners": [str], "roles": [str]},
+        description="Details for the new pack",
+    ),
+    location="body",
+    required=True,
+)
+@doc.produces(
+    {"data": {"name": str, "owners": [str], "pack_id": str, "roles": [str]}},
+    description="Successfully created a new pack",
+)
+@doc.response(
```

Comment:
any functions calling `validate_fields` also throws a 400 code and should include:
```
@doc.response(
    400, {"message": str, "code": int}, description="Bad Request: Improper JSON format."
)
```

Reply:
Added the 400 response to the endpoints that use validate_fields()